changes/bug33375

```diff
@@ -0,0 +1,4 @@
+  o Minor bugfixes (coding best practices checks):
+    - Stop advertising an IPv6 exit policy when DNS is broken for IPv6 by
+      marking the relay descriptor dirty for 24 hours. Fixes bug 33375;
```

Comment:
This changes file doesn't describe what the patch actually does.

Maybe delete "for IPv6 by marking the relay descriptor dirty".